File: src/vs/workbench/contrib/issue/common/commands.ts
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation.
 *  Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

export const OpenIssueReporterActionId = 'workbench.action.openIssueReporter';

export interface OpenIssueReporterArgs {
	readonly extensionId?: string;
	readonly issueTitle?: string;
	readonly issueBody?: string;
}

File: cli/src/commands.rs
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

mod context;

pub mod agent;
pub mod agent_host;
pub mod agent_kill;
pub mod agent_logs;
pub mod agent_ps;
pub mod agent_stop;
pub mod args;
pub mod output;
pub mod serve_web;
pub mod tunnels;
pub mod update;
pub mod version;
pub use context::CommandContext;
